tox.ini

```diff
@@ -1,11 +1,11 @@
 [tox]
 envlist =
-    py37,
+    py{37,38,39},
     py37-flake8
```

Comment:
Not a suggestion, just a question: should we start to prefer 3.9 as the Python version we use with flake8?